### PR TITLE
Completed task

### PR DESCRIPTION
**Pull request by lilarin**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
 
 
 def format_linter_report(linter_report: dict) -> list:
-    # write your code here
-    pass
+    return [
+        {
+            "errors": [format_linter_error(err) for err in errors],
+            "path": path,
+            "status": "failed" if errors else "passed"
```
Comment (Oleg-s-s) [inline app/main.py]: Same as 13 -15 lines, make your code dry

**Review by Oleg-s-s — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 1 above)
Comment (lilarin) [reply app/main.py]: @Oleg-s-s I'm not really clear on what exactly needs to be changed

Comment (lilarin) [reply app/main.py]: Ah, I see what you mean, I forgot to reuse the code

Comment (Oleg-s-s) [reply app/main.py]: DRY = Dont Repeat Yourself

Comment (Oleg-s-s) [reply app/main.py]: https://blog.devgenius.io/the-dry-principle-in-python-boosting-efficiency-in-machine-learning-c8cf4fded736

**Review by Oleg-s-s — APPROVED** [review]
:+1:

**Pull request closed without merging**